compiler/modules/front/lib.bal

```diff
@@ -23,7 +23,7 @@ function getLangLibFunction(string mod, string func) returns bir:FunctionSignatu
 }
 
 final readonly & map<bir:FunctionSignature> ioLibFunctions = {
-    println: { paramTypes: [t:TOP], returnType: t:NIL }
+    println: { paramTypes: [t:TOP], returnType: t:NIL, restParamType: t:TOP }
```

Comment:
Last member of paramTypes is supposed to be a list, when restParamType is non-nil.

Reply:
fixed